dpctl/tensor/libtensor/include/utils/sycl_utils.hpp

```diff
@@ -154,6 +154,30 @@ T custom_reduce_over_group(const GroupT &wg,
     return sycl::group_broadcast(wg, red_val_over_wg);
 }
 
+template <typename T, typename GroupT, typename LocAccT, typename OpT>
+T custom_inclusive_scan_over_group(const GroupT &wg,
+                                   LocAccT local_mem_acc,
+                                   const T local_val,
+                                   const OpT &op)
+{
+    auto local_id = wg.get_local_id(0);
+    auto wgs = wg.get_local_range(0);
```

Comment:
```suggestion
    const std::uint32_t local_id = wg.get_local_id(0);
    const std::uint32_twgs = wg.get_local_range(0);
```

And the `for` loop private variable need to also be `std::uint32_t`.